json-rpc/spec.json

```diff
@@ -1774,6 +1789,25 @@
           ]
         }
       },
+      "AccessListResult": {
+        "name": "accessListResult",
+        "description": "Returns an accesslist, the gas used and an internal error if available",
+        "schema": {
+          "title": "ALResult",
+          "type": "object",
+          "properties": {
+            "accessList": {
+              "$ref": "#/components/schemas/AccessList"
+            },
+            "gasUsed": {
+              "$ref": "#/components/schemas/Integer"
+            },
+            "vmErr": {
```

Comment:
Is this field optional? If an error is returned, is `accessList` and `gasUsed` omitted?